src/document/DocumentCommandHandlers.js

```diff
@@ -159,8 +159,11 @@ define(function (require, exports, module) {
             windowTitle         = brackets.config.app_title,
             currentlyViewedFile = MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE),
             currentlyViewedPath = currentlyViewedFile.fullPath,
-            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "";
-
+            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "",
+            filepathDisplayPref = PreferencesManager.get('showFullPathToFile');
```

Comment:
Currently showFullPathToFile does not match with the name defined for preference 'showFullFilePathInTopBar'.

Reply:
Okay. And I've changed the name as per your commented suggestion above.